src/modules/cluster/import-cluster/import-cluster.js

```diff
@@ -6,32 +6,88 @@
     app.controller("importClusterController", importClusterController);
 
     /*@ngInject*/
-    function importClusterController($rootScope, $state, utils, config) {
-        var vm = this;
-
-        vm.heading = "Import Cluster";
-        vm.isShowImportButton = true;
-        vm.importFlows = [];
-        vm.selectedFlow = {};
-        vm.setImportClusterInfo = setImportClusterInfo;
-        vm.callBack = callBack;
-
-        utils.getObjectWorkflows().then(function(importFlows) {
-              vm.importFlows = importFlows;
-        });
-
-        function setImportClusterInfo(flow) {
-            vm.heading = flow.name;
-            vm.isShowImportButton = false;
-            vm.selectedFlow = flow;
+    function importClusterController($rootScope, $state, $stateParams, utils) {
+        var vm = this, hostList = [], hostDetail = {};
+        
+        vm.step = 1;
+        vm.detectedClusters = [];
+        vm.init = init;
+        vm.importCluster = importCluster;
+        vm.importCancel = importCancel;
+        vm.viewTaskProgress = viewTaskProgress;
+        vm.selectCluster = selectCluster;
+        vm.init();
```

Comment:
@kamleshverma1 Put only those functions on controller instance which needs to be accessed from html. Please remove init from vm instance.